doc/CONTRIBUTING.md

```diff
@@ -28,3 +28,72 @@ For both owners and contributors, contribution guidelines save time and hassle c
 7. Appreciate your team as you would want to be appreciated for all your hard work. It creates a safe environment for spontenuety and highten a sense of wellbeing of the entire team.
 
 8. No pressure, just work hard and do your best.
+
+New Suggestions by Charlene Grant (draft edit added here)
+
+a) speak up and build your confidence to do more of expressing your views 
+
+b) find another member like Tanya to confide in and speak on your behalf 
+
+c) Never feel pressured to stay but try to see the process through you might suprise yourself 
+
+d) Always try to change a miserable attitude to a winning attitude
+
+e) process it, brush it off, fix your crown and keep moving forward.
+
+# Pull Request Template
+(Feat): This is a new feature
+(Fix): I am fixing something
+(Doc): Some new documentation
+
+## Description
+
+:question: Please include a summary of the change(s) a small paragraph in your own words is fine.
+
+
+:question: Break Down of Task, steps taken.
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+## Type of change
+
+:question: Please tick all relevant options
+
+- [ ] **Bug fix** (non-breaking change which fixes an issue)
+- [ ] **New feature** (non-breaking change which adds functionality)
+- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
+- [ ] **This change requires a documentation update**
+
+-[ ] ...
+
+:question: Have you used any tutorials or done any reading that helped you to complete these changes and would be of benefit to the group? If yes you can add links here. Great for our growth but also so we can help if needed.
+
+## Issue Addressed
+
+:question: This pull request is in reponse to **issue** #
+
+:question: Does it require the install of any other dependencies apart from the standard node_modules when a **npm install** is run? If yes please list here.
+
+## Has This Been Tested?
+
+:question: Please describe any tests used in a way that the rest of the group could perform the same test. 
+
+## Checklist:
+
+:question: For peace of mind.
+
+- [ ] My code follows the guidelines of this project
+- [ ] I have used appropriate and relevant pull request title
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation or I have not needed to
+- [ ] My changes generate no new warnings
+- [ ] I have checked my code and corrected any misspellings
+- [ ] Linked issues
+- [ ] Assigned team or specific members to review.
```

Comment:
This section does not belong here,  it was just those 3 lines to be added as Tanya suggested.

maybe the sentence would read- 
When deciding a title/name for your pull request please prefix with: 
a. (Feat): This is a new feature
b. (Fix): I am fixing something
c. (Doc): Some new documentation
for example "fix.folder-structure" this outlines what you are doing and what you have done. it is short and sweet, to the point and informative. It will make documenting and revisiting code in the future easy to read and understand,

Or something similar.

Reply:
I haven't made any changes. I was just testing out pull request. The contributing guidelines look really good